apps/src/JSInterpreter.js

```diff
@@ -434,8 +435,9 @@ JSInterpreter.prototype.executeInterpreter = function (firstStep, runUntilCallba
     var err = safeStepInterpreter(this);
     if (!err) {
       var nodeType = this.interpreter.stateStack[0].node.type;
+      this.atInterstitialNode = INTERSTITIAL_NODES.hasOwnProperty(nodeType);
```

Comment:
does this only impact revisiting interstitial nodes? doesn't it set it the first time as well (which will impact the check above)?